rotkehlchen/data_handler.py

```diff
@@ -95,6 +102,7 @@ def unlock(
         self.user_data_dir = user_data_dir
         self.logged_in = True
         self.username = username
+        self.password = password
```

Comment:
hehe, this is a bit bad. But as we discussed in discord probably to read this you would have to have access to the users system and even then it's hard. But then you can do lots of bad things.